packages/serverpod/lib/src/database/table_relation.dart

```diff
@@ -33,6 +33,11 @@ class TableRelation {
     return _tableRelationEntries.last.foreignField.table.tableName;
   }
 
+  /// Name of the last field to be joined.
+  String get lastJoiningFieldQueryAlias {
```

Comment:
Chore: Add a unit test for these strings in `table_relation_test.dart`

Could you please add a unit test for the new method in that file, that helps us keeping track of what is expected from these in different situations.